core-bundle/src/Exception/InvalidResourceException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+class InvalidResourceException extends \InvalidArgumentException
```

Comment:
maybe add a comment that this is supposed to represent a DBAFS resource?

Reply:
It's not limited to DBAFS resources but also applies to file paths. For example see: https://github.com/contao/contao/blob/e589b9701386ba5bd141443db2ca1f95f95137c6/core-bundle/src/Image/Studio/FigureBuilder.php#L200-L202